tests/ci_reset_hw.py

```diff
@@ -0,0 +1,31 @@
+from panda import Panda, PandaDFU
+from libs.resetter import Resetter  # pylint: disable=import-error
+
+r = Resetter()
```

Comment:
always guard with `if __name__ == '__main__':`. you don't want this to run at import time